src/web/README.md

```diff
@@ -1,12 +1,15 @@
-# Webserver subsystem
+# Web server subsystem
 
-This folder contains all of the classes for running the webserver.
+This folder contains all of the classes for running the web server.
+
+The web server subsystem:
 
-The webserver subsystem
 - Handles JSON-RPC and websocket requests.
+
 - Supports SSL if a cert and key file are specified in the config.
+
 - Handles all types of requests on a single port.
 
-Each request is handled asynchronously using boost asio.
+Each request is handled asynchronously using [Boost Asio](https://www.boost.org/doc/libs/1_84_0/doc/html/boost_asio.html).
```

Comment:
We are at Boost 1_82_0 atm. so maybe let's update the link.. I don't think there is a way to link to the actually used boost (specified via `conanfile.py`) so best we can do is use the correct version's link here.